packages-test.dhall

```diff
@@ -0,0 +1,13 @@
+let upstream =
```

Comment:
Was there a specific reason to change the way test environment is set up?

Reply:
I ran into some issues because of the cyclic dependency. Guess this could be reverted now.

Reply:
reverted to previous setup